src/test/java/core/basesyntax/service/CreateReportServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.impl.CreateReportImpl;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CreateReportServiceTest {
+    private static final String HEADER_FRUIT = "fruitTransaction,";
+    private static final String HEADER_QUANTITY = "quantity";
+    private static final String BANANA_REPORT = "banana 152";
+    private static final String APPLE_REPORT = "apple 90";
+    private static CreateReportService createReport;
+
+    @BeforeClass
+    public static void beforeClass() {
+        createReport = new CreateReportImpl();
+    }
+
+    @Test
+    public void createReport_validData_ok() {
+        String expectedReportString = expectedReport();
```

Comment:
This will be easier to read and understand. And why "fruitTransaction" in the header???
```suggestion
        String expectedReportString = "fruit,quantity" + System.lineSeparator()
+ "banana 152" + System.lineSeparator()
+ "apple 90";
```